src/install/config/kubernetes.yaml

```diff
@@ -0,0 +1,70 @@
+agentName: kubernetes
+agentType: integration
+title: 'Kubernetes monitoring integration'
+metaDescription: 'Install the Kubernetes integration'
+introFilePath: 'src/install/kubernetes/intro.mdx'
+appInfo: 
+  - optionType: dropdown1
+    label: ''
+    placeholder: 'Select your environment'
+    options: 
+      - value: 'guided'
+        displayName: 'Guided install'
+        logo: 'k8'
+      - value: 'helm'
+        displayName: 'Helm'
+        logo: 'k8'
+      - value: 'manifest'
+        displayName: 'Manifest'
+        logo: 'k8'
+
+steps:
+  - filePath: 'src/install/kubernetes/appInfo.mdx'
+  - filePath: 'src/install/kubernetes/default-requirements.mdx'
+    overrides: 
+        - filePath: 'src/install/kubernetes/guided/requirements-guided.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'guided'
+        - filePath: 'src/install/kubernetes/helm/requirements-helm.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'helm'
+  - filePath: "src/install/kubernetes/install.mdx"
+    overrides: 
+        - filePath: 'src/install/kubernetes/guided/install-guided.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'guided' 
+        - filePath: 'src/install/kubernetes/helm/install-helm.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'helm' 
+        - filePath: 'src/install/kubernetes/manifest/install-manifest.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'manifest'       
+  - filePath: 'src/install/kubernetes/configuration.mdx'
+    overrides: 
+        - filePath: 'src/install/kubernetes/guided/configuration-guided.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'guided' 
+        - filePath: 'src/install/kubernetes/helm/configuration-helm.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'helm' 
+        - filePath: 'src/install/kubernetes/manifest/configuration-manifest.mdx'
+          selectedOptions: 
+            - optionType: dropdown1
+              options: 
+                - value: 'manifest'
+  - filePath: 'src/install/mongodb/find-data.mdx'
```

Comment:
changing these file paths to use `/kubernetes/` instead of `/mongodb/` fixed the issue. Is there a reason we need mongodb here?

Reply:
ouch! My bad.... a clumsy copy and paste. Thank you, @clarkmcadoo !